package.json

```diff
@@ -23,7 +23,8 @@
     "globals": {
       "__DEV__": true
     },
-    "testRegex": ".+\\.test\\.js$"
+    "testRegex": ".+\\.test\\.js$",
+    "testURL": "http://localhost"
```

Comment:
Little out of scope, but this fixes an issue I was running into 😅 

Issue: https://github.com/jsdom/jsdom/issues/2304
Fix: https://stackoverflow.com/a/41710136